model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -464,6 +464,49 @@ def _register_computed_fields(self):
             pairs=(("u_cell_1", "u_cell_2"), ("v_cell_1", "v_cell_2")),
         )
         self.register_provider(tangent_cell_wrapper)
+        cartesian_vertex_centers = factory.FieldOperatorProvider(
+            # TODO: ..._on_vertices?
+            func=math_helpers.geographical_to_cartesian_on_vertex.with_backend(self.backend),
```

Comment:
Should this be
```suggestion
            func=math_helpers.geographical_to_cartesian_on_vertices.with_backend(self.backend),
```
? The edge and cell functions are called `on_edges` and `on_cells`, respectively.